runtime/pom.xml

```diff
@@ -228,7 +228,8 @@
                 </executions>
                 <configuration>
                     <outputDirectory>${project.parent.basedir}/target/coverage-reports</outputDirectory>
-                    <dataFile>${project.parent.basedir}/jacoco.exec</dataFile>
+                    <dataFile>${project.parent.basedir}/target/coverage-reports/jacoco.exec</dataFile>
+                    <skip>true</skip>
```

Comment:
hi @jinyangyang222 instead of directly using the "skip" flag.. can you please link it to a maven property?
Similar to our current flag `docker.skip`. Then it is more clear in the Maven command, what exactly we are skipping or not skipping.

Reply:
fixed